content/3_Silver/Priority_Queues.problems.json

```diff
@@ -112,6 +150,19 @@
         "kind": "autogen-label-from-site",
         "site": "CF"
       }
+    },
+    {
+      "uniqueId": "cf-2085D",
+      "name": "Serval and Kaitenzushi Buffet",
+      "url": "https://codeforces.com/contest/2085/problem/D",
+      "source": "CF",
+      "difficulty": "Hard",
```

Comment:
isn't this problem pretty much the same as William and Robot, why is the difficulty different? 🤔

Reply:
it's more complex no? I think its harder

Reply:
I don't agree (my solution codes look nearly the same), but fine to leave as is